cmd/listGsPlans.go

```diff
@@ -0,0 +1,81 @@
+// Copyright © 2018 Infostellar, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"time"
+	"github.com/infostellarinc/stellarcli/pkg/plan"
+	"fmt"
+)
+
+const (
+	// Time format used to parse "after" and "before" flags.
+	timeFormat = "2006-01-02 15:04:05"
+	// Default time range used when end time is not specified.
+	defaultDurationInDays = 31
+	// Maximum value of duration in days.
+	maxDurationInDays = 31
+)
+
+var (
+	flgAOSAfter  string
+	flgAOSBefore string
+	flgDuration  uint8
+)
+
+// listGSPlansCmd represents the ground station command
+var listGSPlansCmd = &cobra.Command{
+	Use:   "list-plans [Ground Station ID]",
+	Short: "Lists plans on a ground station.",
+	Long:  `Lists plans on a ground station. Plans having AOS between the given time range will be returned.`,
```

Comment:
Let's go ahead and add one more comment

When run with default flags, plans in the next 31 days are returned.